js/components/home.tsx

```diff
@@ -54,11 +59,14 @@ export const Home = (): ReactElement => {
       </section>
       <section className="max-w-lg mx-auto px-2 py-5">
         <p className="font-semibold mb-3">Today&apos;s sign ins</p>
-        <List line="blue" />
+        <List key={signInCount} line="blue" />
       </section>
 
       <OperatorSignInModal
         show={modalOpen}
+        onComplete={() => {
+          setSignInCount(signInCount + 1);
```

Comment:
**suggestion:** this is a case where you can pass a callback to a setter instead of a new value:
```suggestion
          setSignInCount((count) => count + 1);
```

I don't think it makes a huge difference here but I think it's vaguely more idiomatic and can be nice in certain situations (for instance, inside of a `useEffect` it can avoid having to add the old value as a depndency).

Reply:
nice! fixed fd3dcde